tests/ResarchStation/Domain/SpaceResearchStation/SpaceResearchStationTest.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Tests\ResarchStation\Domain\SpaceResearchStation;
+
+use App\DomainModel\EarthScientistDomain;
+use App\DomainModel\SpaceResearchStation;
+use App\DomainModel\SpaceScientist;
+use App\Exception\WrongScientistTypeException;
+use PHPUnit\Framework\TestCase;
+use function count;
+
+class SpaceResearchStationTest extends TestCase
+{
+    private SpaceResearchStation $researchStation;
+
+    /** @test */
+    public function try_to_add_wrong_scientist_type_should_throw_error(): void
+    {
+        $this->expectException(WrongScientistTypeException::class);
+
+        $this->researchStation = new SpaceResearchStation(
+            1,
+            70,
+            100,
+            10000.05,
+            12,
+            12.5,
+            89,
+            12,
+            [],
+            [],
+            [],
+            false
```

Comment:
Nie wiem, czy dla ciebie to jest czytelne, ale polecam w takich przypadakch uzywac:

https://stitcher.io/blog/php-8-named-arguments

Reply:
Sprytne